libs/components/forms/src/lib/modules/radio/radio-group-id.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@angular/core';
+
+import { BehaviorSubject, Observable } from 'rxjs';
+
+interface RegisterArgs {
```

Comment:
Since `SkyRadioGroupIdService` isn't part of the public API, I wouldn't worry about making an `args` interface. That's more for future-proofing public APIs for backwards compatibility.